cluster/bin/cluster-env

```diff
@@ -23,7 +23,13 @@ if [ ! -f "$HOME/.ssh/config" -a ! -f "$HOME/.ssh/cluster" ]; then
     echo "Configuring SSH for cluster access"
     install -d -m 700 $HOME/.ssh
     ssh-keygen -t rsa -f $HOME/.ssh/cluster -N '' -C "Warewulf Cluster key" > /dev/null 2>&1
+    if [ -f $HOME/.ssh/authorized_keys -a ! -w $HOME/.ssh/authorized_keys ]; then
```

Comment:
This is a tiny nit, but I generally prefer this style as I believe shellcheck will complain

```
if [ -f $HOME/.ssh/authorized_keys ] && [ ! -w $HOME/.ssh/authorized_keys ]; then
```

Reply:
Changed. May eventually go through with shellcheck at some point and clean it up the scripts to make it happy. If you wanted you could it could be added into CI to be sure we keep them consistent.

Reply:
Yea I've wanted to enforce Shellcheck for a while, but lack the cycles.

Reply:
I'll see if I can't shift some to get that started then here.